R/count-tally.R

```diff
@@ -61,23 +63,31 @@
 #' # mutate() instead of summarise
 #' df %>% add_count(gender, wt = runs)
 #' df %>% add_tally(wt = runs)
-count <- function(x, ..., wt = NULL, sort = FALSE, name = NULL) {
+count <- function(.data, ..., wt = NULL, sort = FALSE, name = NULL, x = deprecated()) {
+  if (lifecycle::is_present(x)) {
+    lifecycle::deprecate_warn("1.0.8", "count(x)", "count(.data)")
+    return(count_dispatch(
+      .data = x, !!enexpr(.data), ..., wt = !!enquo(wt),
+      sort = sort, name = name
```

Comment:
Can we hug the arguments here and elsewhere?

```suggestion
      .data = x, {{ .data }}, ..., wt = {{ wt }},
      sort = sort, name = name
```